README.md

```diff
@@ -12,7 +12,7 @@ This is the shared course code for:
 ## Example
 Run tests for [LC100_isSameTree](exercises/LC100_isSameTree)
 
-1) `cd` into the [exercises](exercises) folder
+1) `cd` into the [trees](trees) folder
```

Comment:
You're videos still say to descend into exercise fwiw. Obviously easy to figure out but 🤷‍♂️